pyhessian/hessian.py

```diff
@@ -260,8 +260,8 @@ def density(self, iter=100, n_v=1):
                     T[i, i + 1] = beta_list[i]
             a_, b_ = torch.linalg.eig(T)
 
-            eigen_list = a_
-            weight_list = torch.pow(b_, 2)
+            eigen_list = a_.real
+            weight_list = torch.pow(b_[0,:], 2)
```

Comment:
Thank you very much for the catch. Can you please also rename a_ and b_ to eigenvalues and eigenvectors to make this more clear (similar to https://github.com/amirgholami/PyHessian/pull/22/files)? Thanks again

Reply:
Updated with renamed variables!